app/views/articles/index.html.erb

```diff
@@ -32,11 +32,12 @@
       data-feed="<%= params[:timeframe] || "base-feed" %>"
       data-articles-since="<%= Timeframer.new(params[:timeframe]).datetime&.iso8601 %>">
 
+    <h1 class="visually-hidden-header"><%= community_name %></h1>
```

Comment:
One outstanding question I have is about this h1 that should only live on the homepage. It currently sits outside of `<main>` so it can be rendered before the sidebar, which contains headings that throw off the order as the h1 should come first. Because this heading needs to live inside of a landmark element, I tried conditionally rendering an h1 in `_logo.html.erb` by checking for the homepage and rendering different markup, but it didn't update on the client-side when clicking through pages. I could use some help getting a solution figured out! Maybe `content_for` in Rails to conditionally render in the header on the homepage?

Reply:
> I tried conditionally rendering an h1 in _logo.html.erb by checking for the homepage and rendering different markup, but it didn't update on the client-side when clicking through pages.

Yeap, this may not work as you expect since we do lots of weird things around caching and serviceworkers to speed up loading pages... It's pretty tough to overcome TBH, not sure if worth the effort in this case...

Actually... What do you think about removing this element entirely? It's kinda pointless, right? It was pointless earlier when it was saying "Articles" because of its placement (as well as the fact that we do have normal label right below it saying "Posts") and it is a bit pointless now too when it returns community name.. users probably know where they are (it's basically name of the site) and also that element is pretty close to Header where we also have logo and community name so the screen reader would read it anyway, right?

Reply:
That makes sense about the caching, and is consistent with what I saw in the application! Though, this element is not pointless as the community name is only part of the heading structure if we put it in a heading...and that seems important, at least on the homepage. The site logo in the header is a link and won't end up in the heading structure unless changed. 

The easiest way forward, I think, is it put the homepage h1 back into the `<main>` element and let the couple of headings in the sidebar come before it. Surprisingly, this resource from the W3C WAI has an example where the h1 doesn't come first: https://www.w3.org/WAI/tutorials/page-structure/headings/

Therefore, I'll update my PR so the homepage can include a more useful h1.